File: 프로그래머스/0/120848. 팩토리얼/팩토리얼.java
class Solution {
    public int solution(int n) {
        int count = 0;
        int num = 1; 
        for (int i = 1; i <= 3628800; i++) {
            num *= i;
            if (num > n) {
                break;
            } else {
                count++;
            }
        }
        
        int answer = count;
        return answer;
    }
}

File: 백준/Bronze/10872. 팩토리얼/팩토리얼.c
#include <stdio.h>

int main(void) {
    int num;
    int answer = 1; 
    scanf("%d", &num);
    if (num == 0) {
        printf("%d", answer);
    } else {
        for (int i = 1; i <= num; i++) {
            answer *= i;
        }
        printf("%d", answer);
    }
    return 0; 
}